data/common.yaml

```diff
@@ -0,0 +1,28 @@
+rkhunter::allowhiddenfile:
```

Comment:
I didn't include some of those defaults because they were files that didn't exist on a bare CentOS install, nor a simp load, so I didn't want to whitelist something that could pop up on a system masquerading as something good when it is not actually supposed to be there for their system.

Reply:
Some of the others came from their sourceforge https://sourceforge.net/p/rkhunter/rkh_code/ci/master/tree/files/rkhunter.conf